src/composites/FeaturesContainer/Overview/OverviewDropLists.tsx

```diff
@@ -0,0 +1,62 @@
+// import { Component, ReactNode } from "react";
+import { OverviewData, ListNode } from './index.ts';
+import * as Accordion from "@radix-ui/react-accordion";
+import { Theme, Box, Text, Strong } from '@radix-ui/themes';
+import { DotFilledIcon } from '@radix-ui/react-icons';
+import "./Overview.css"
+
+export const OverviewDropList = (risk:string, section:string, props:OverviewData) => {
```

Comment:
refer to my last review, this needs to be a component

Reply:
Addressed from previous comments